app/code/Magento/Checkout/view/frontend/web/js/sidebar.js

```diff
@@ -77,8 +77,10 @@ define([
                         confirm: function () {
                             self._removeItem($(event.currentTarget));
                         },
-                        always: function (event) {
-                            event.stopImmediatePropagation();
+
+                        /** @inheritdoc */
+                        always: function (e) {
```

Comment:
Could you describe why this code was added? I can't find this change in original PR.

Reply:
Ah, just in order to be the same across all release lines. It's a good idea.